subsys/bluetooth/controller/Kconfig

```diff
@@ -750,4 +750,18 @@ config BT_MAYFLY_YIELD_AFTER_CALL
 	  If set to 'n', all pending mayflies for callee are executed before
 	  yielding
 
+config BT_CTLR_USER_EXT
```

Comment:
Here's how I would do it:

```
config BT_CTLR_USER_EVTS
        def_bool (BT_CTLR_USER_EVT_RANGE != 0)
        # omit prompt to signify a "hidden" option
        help
          This hidden option specifies that user events are enabled.

config BT_CTLR_USER_EVT_RANGE
	int "Range of event contants reserved for proprietary event types"
	default 0
	range 0 10
	help
	  Number of event types reserved for proprietary use. 
          If 0, user events are disabled.
```

Reply:
I would prefer to keep BT_CTLR_USER_EXT open for other use

Reply:
But you want to keep it as a global on/off switch of everything that is a user extension?